source/scss/_library/_objects.navs.scss

```diff
@@ -224,3 +224,59 @@
     }
   }
 }
+
+
+/**
+ * Read More navigation
+ */
+.c-read-more-nav {
+  padding: $space-and-half $space;
+  border: 2px dashed $c-gray;
+  text-align: center;
+
+  @include media('<=medium') {
+    margin-left: -$space;
+    margin-right: -$space;
+    width: 100vw;
```

Comment:
can these kind of margin-busting attributes get added to a special container class? I'm worried this will too tightly couple "read more" to its current position